snapshots/cardano-1.25.1-GHC-8.10.4.yaml

```diff
@@ -0,0 +1,155 @@
+name: cardano-1.25.1-GHC-8.10.4
+
+resolver: lts-17.4
+
+packages:
+- base16-0.1.2.1
+- base16-bytestring-1.0.1.0
```

Comment:
Maybe https://www.stackage.org/lts-17.6/package/base16-bytestring-0.1.1.7

Reply:
This one needs to be base16-bytestring-1.0.1.0, due to breaking changes. cardano-base already updated to the latest.